nim/game.py

```diff
@@ -92,6 +103,18 @@ def __init__(self):
         self.chaos = Chaos()
 
     def new_game(self, min, max, piles):
+        """
+        Logic of new game dealt with here.
+        Uses min, max & piles to return a
+        list of piles
```

Comment:
NIT: you don't need to split the line as long as it's not more than 80 characters long (and if you forget, tox should fail). You have other occurrences in your PR where you did this, I won't comment on all of them but you should update them.